### PR TITLE
Revive MSP message for mag declination

### PR DESCRIPTION
**Pull request by haslinghuis**
- Firmware  https://github.com/betaflight/betaflight/pull/13232

- 90 means `+9.0 degrees` declination 
- 3510 means `-9.0 degrees` declination
- Units are `decidegrees` afaik. 
- Negative is `3600 - declinationDeg * 10`

Setting is allowed for firmware API 1.46 and up if the magnetometer is active.

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
LGTM. As always I will appreciate a screen capture, but the code looks good.

**Review by nerdCopter — APPROVED** [review]
* approving untested, as i see firmware part merged. skimmed code, looks good.

**Pull request merged**
